src/Keboola/S3Extractor/ConfigDefinition.php

```diff
@@ -57,4 +69,29 @@ protected function getParametersDefinition(): ArrayNodeDefinition
        // @formatter:on
         return $parametersNode;
     }
+
+    private function addValidate(NodeDefinition $definition): void
+    {
+        $definition->validate()->always(function ($item) {
+            if (!isset($item['loginType']) || $item['loginType'] === 'credentials') {
+                if (!isset($item['accessKeyId'])) {
+                    throw new InvalidConfigurationException('The child node "accessKeyId" at path "root.parameters" must be configured.');
+                }
+                if (!isset($item['#secretAccessKey'])) {
+                    throw new InvalidConfigurationException('The child node "#secretAccessKey" at path "root.parameters" must be configured.');
```

Comment:
Je tato validácia nejak pokrytá testami?

Reply:
nekoukal jsem - tahle validace už tam byla tak doufám že jo - radši zkontroluji

Reply:
tak to otestovaný nebylo, pčidal jsem v commitu:
https://github.com/keboola/aws-s3-extractor/pull/61/commits/c99deb6bc761ad93a058ef197a02249838988981